tests/unittests/patches_tests.py

```diff
@@ -102,13 +102,13 @@ def test_to_patches(self):
         self.assertSetEqual(set(indexes), default_indexes)
 
         with self.assertRaises(ValueError):
-            view.drop_index("id")  # can't drop default index
+            view.drop_index("id", safe_mode=False)  # can't drop default index
```

Comment:
When we set the env var `FIFTYONE_DATABASE_SAFE_MODE` to False, we do not need to set this flag any more :)